Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,96 @@
+﻿using System;
+
+namespace BootCamp.Chapter
+{
+    class Lesson3
+    {
+        public static void Demo()
+        {
+            bool calculateAgain = true;
+
+            while (calculateAgain)
+            {
+                Console.WriteLine("BMI Calculator");
+
+                string name = GetString("Enter your name: ");
+                int age = GetInt("Enter your age: ");
+                float weight = GetFloat("Enter your weight(kg): ");
+                float height = GetFloat("Enter your height(cm): ");
```

Comment:
Get is a rather bad name for what those functions do. Prompt (as in Checks.cs) are a proper name for requested input.